iOS Client/PodcastsController/PodcastDetailViewController.swift

```diff
@@ -71,6 +90,25 @@ class PodcastDetailViewController: UIViewController, StoreSubscriber {
         }
     }
     
+    func animateImage(size: CGSize, duration: CFTimeInterval) {
+        var shadowRect = imageView.bounds
+        var fromRect = CGRect(x: shadowRect.origin.x, y: shadowRect.origin.y, width: shadowRect.width + 10, height: shadowRect.height + 10)
+        shadowRect.size = CGSize(width: size.width + 10, height: size.height + 10)
+        let shadowAnimation = CABasicAnimation(keyPath: "shadowPath")
```

Comment:
not 100% if it'll work but try the `#keypath(UIImage.shadowPath)` pattern (not sure if that's the right path, maybe it's UIImage.layer.shadowPath or something )

Reply:
it will avoid stringly typed code trap

Reply:
I'll check it out not exactly sure how `#keypath` works but I'll give it a shot